SDK: Codes for Requesting to display rewarded ads to js bridge. (#43)
rakutentech/js-miniapp: js-miniapp-sdk/README.md

	});
```

### 4. Show Ads

Mini App SDK allows to display ads upon requesting from Mini apps.
This requires the mini-app to have obtained/enrolled the required ad units with the host app.
Currently two ad types are supported,
1. Interstitial
2. Rewarded

> | Interstitial |
> | ------------ |

Reviewer: This will make this formatted as a table, which looks a little weird as a section header. Maybe just make it bold instead?

```suggestion
**Interstitial**
```

Author: Removed table design!

	});
```

### 4. Show Ads

> Mini App SDK allows to display ads upon requesting from Mini apps requesting from Mini apps with ad unit id.

Reviewer: ```suggestion
Mini App SDK allows you to display ads upon requesting from a Mini App with an ad unit id.
```

> This requires to pass in ad unit id to the host app in order to load the ad associated with the ad unit id.

Reviewer: ```suggestion
This requires you to first load an Ad by passing an ID. You can then display that ad by passing the same ID which was loaded.

Note that typically you should load your Ads at some point earlier than you intend to use them, such as at App launch time. You can also pre-load multiple Ads by calling `MiniApp.loadInterstialAd` or `MiniApp.loadRewardedAd multiple times.
```

Currently two ad types are supported,
1. Interstitial
2. Rewarded

**Interstitial**
The results of the display `Interstitial` ad action is captured as [InterstitialAdResponse](src/types/responseTypes/interstitial/index.ts).

```javascript

> const adUnitID = 'xxx-xxx-xxxxxxxxxxxxx';
> miniApp.showInterstitialAd(adUnitID)
> 	.then(response => {
> 		console.log(response); // Successful response
> 	}).catch(error => {
> 		console.error(response); // Error
> 	});

Reviewer: I think we need to include the complete example of loading the Ad first and then displaying the Ad.

```suggestion
const adUnitID = 'xxx-xxx-xxxxxxxxxxxxx';

miniApp.loadInterstitialAd(adUnitID)
    .then(response => {
        miniApp.showInterstitialAd(adUnitID)
            .then(response => console.log(response) )
            .catch( error => console.error(response) );
    })
    .catch( error => console.error(response) );
```

```
**Rewarded**
The results of the display `Reward` ad action is captured as [RewardedAdResponse](src/types/responseTypes/rewarded/index.ts).

```javascript
const adUnitID = 'xxx-xxx-xxxxxxxxxxxxx';

> miniApp.showRewardedAd(adUnitID)

Reviewer: Same comment here as on `showInterstitialAd`: we should show the complete example of loading the ad first.

.then(response => {
		console.log(response); // Successful response
	}).catch(error => {
		console.error(response); // Error
	});
```

> ### 5. Load Ads

Reviewer: This section is displayed in the wrong order: Ads must be loaded first and then displayed. The `load` and `show` functionality must be used together so I think the `Load Ads` section should be removed and combined with the `Show Ads` section. See my comments above.

	});
```

### 4. Show Ads

Mini App SDK allows you to display ads upon requesting from a Mini App with an ad unit id.

> This requires you to first load an Ad by passing an ID. You can then display that ad by passing the same ID which was loaded.

Reviewer 2: ```suggestion
This requires you to first load an Ad by passing an ID. You can then display an Ad in the Ad Unit by passing the same ID which was used for loading.
```
Does it describe the flow better?

		console.error(response); // Error
	});
```

> | Rewarded |
> | -------- |

Reviewer: Same here:

```suggestion
**Rewarded**
```

Author: Removed!
